### PR TITLE
fix(av-moderation): buttons for participants pane

### PR DESCRIPTION
**Pull request by gabiborlea**
Before
Here should be a Ask to Unmute button
Screenshot 2023-02-28 at 17 29 24
Here the the participant has the video muted and mic on, the moderator does not have the allow video option
Screenshot 2023-02-28 at 17 29 40
Also this is the remote menu on the tiles, no allow video:
Screenshot 2023-02-28 at 17 30 15

After:
Screenshot 2023-02-28 at 17 27 45
Screenshot 2023-02-28 at 17 28 00
Screenshot 2023-02-28 at 17 28 11
Screenshot 2023-02-28 at 17 28 22

Also added the stop video quick action:
Screenshot 2023-02-28 at 17 34 44

### CODE REVIEW

**Comment by damencho** [comment]
AudioVideoModerationTest tests is failing, can you check those?

File: react/features/video-menu/components/web/ParticipantContextMenu.tsx [hunk 1]
```diff
@@ -172,6 +172,10 @@ const ParticipantContextMenu = ({
     }
     , [ thumbnailMenu, _overflowDrawer, drawerParticipant, participant ]);
 
+    const isParticipantPane = useMemo(() => !_overflowDrawer && !thumbnailMenu, [ _overflowDrawer, thumbnailMenu ]);
```
Comment (quitrk) [inline react/features/video-menu/components/web/ParticipantContextMenu.tsx:175]: rename to something clearer please

**Comment by gabiborlea** [comment]
Jenkins test this please

**Comment by gabiborlea** [comment]
Jenkins test this please

**Comment by gabiborlea** [comment]
Jenkins test this please

**Review by damencho — APPROVED** [review]

**Pull request merged**